src/Card/CardCarousel/CardCarousel.d.ts

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+
+export interface CardCarouselProps {
+  ariaLabel: string,
+  children: React.ReactNode,
+  title?: React.ReactNode,
+  subtitle?: React.ReactNode,
+  columnSizes?: {
+    xs?: number,
+    sm?: number,
+    md?: number,
+    lg?: number,
+    xl?: number,
+  },
+  hasInteractiveChildren?: boolean,
+  canScrollHorizontal?: boolean,
+  disableOpacityMasks?: boolean,
+  onScrollPrevious?: Function,
+  onScrollNext?: Function,
```

Comment:
Let's clarify that functions do not return anything (for example: `clickHandler?: () => void`)